src/cryptsetup/cryptsetup.c

```diff
@@ -1045,6 +1045,13 @@ static int measure_volume_key(
 #endif
 }
 
+static int notify_external_activation(const char *volume) {
```

Comment:
given that this ultimately just logs, maybe make that clear in the name, i.e. log_external_activation()?

it's what we usually do with logging helpers.

also our logging functions typically generate structured messages, that include an ERRNO= field about the error codes we are operating on. Hence I think it would make sense to use log_info_errno() here, and make this function take the error number as first arg, even if it's always going to be EEXIST here, and even though you are not using %m.

It's just a matter of coding style to propagate errors we have received through to the log subsystem if we log about them, regardless if this is fatal or not.